### PR TITLE
tiup: ducument how to maintain a TiDB cluster using tiup

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: how-to/maintain/tiup-operations.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+此操作会按顺序启动整个 TiDB 集群所有组件（包括 PD、TiDB、TiKV 等组件和监控组件）。
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster start <cluster-name>
+```
+
+> 请将 <cluster-name> 替换成实际的集群名字，若忘记集群名字，可通过 `tiup cluster list` 查看
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:28]:
```suggestion
> 请将 `<cluster-name>` 替换成实际的集群名字，若忘记集群名字，可通过 `tiup cluster list` 查看
```

File: how-to/maintain/tiup-operations.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster list
+```
+
+## 启动集群
+
+此操作会按顺序启动整个 TiDB 集群所有组件（包括 PD、TiDB、TiKV 等组件和监控组件）。
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:20]: 顺序也说一下吧

File: how-to/maintain/tiup-operations.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster display <cluster-name>
+```
+
+## 关闭集群
+
+此操作会按顺序关闭整个 TiDB 集群所有组件（包括 PD、TiDB、TiKV 等组件和监控组件）
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:58]: 顺序说一下吧

File: how-to/maintain/tiup-operations.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+而下面的命令只启动 `1.2.3.4` 和 `1.2.3.5` 这两台机器上的 tidb 组件
+
+```bash
+tiup cluster start test-cluster -R tidb -N 1.2.3.4:4000,1.2.3.5:4000
+```
+
+## 销毁集群
+
+此操作会关闭服务，清空数据目录和部署目录。
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:80]:
```suggestion
此操作会关闭服务，清空数据目录和部署目录，并且无法恢复，请谨慎操作。
```

File: how-to/maintain/tiup-operations.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster stop <cluster-name>
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:85]: stop 写错了吧，destroy

File: how-to/maintain/tiup-operations.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+该命令支持通过 `-R` 和 `-N` 参数来只启动部分组件，例如下面的命令只启动 test-cluster 的 pd 组件
+
+```bash
+tiup cluster start test-cluster -R pd
+```
+
+而下面的命令只启动 `1.2.3.4` 和 `1.2.3.5` 这两台机器上的 pd 组件
+
+```bash
+tiup cluster start test-cluster -R pd -N 1.2.3.4:2379,1.2.3.5:2379
```
Comment (kissmydb) [inline how-to/maintain/tiup-operations.md:39]: 1.2.3.4:2379 只能是 pd，还需要指定 -R pd 么？

Comment (lucklove) [reply how-to/maintain/tiup-operations.md:39]: 2379只是个端口，不一定是 PD，用户配置 tidb 监听在 2379 也是可以的，只是这比较少见

File: how-to/maintain/tiup-operations.md (hunk 5 above)
Comment (lucklove) [reply how-to/maintain/tiup-operations.md:85]: 写错了，sorry

File: how-to/maintain/tiup-operations.md (hunk 6 above)
Comment (lucklove) [reply how-to/maintain/tiup-operations.md:39]: 为了避免误会我把 -R 去掉吧

File: how-to/maintain/tiup-operations.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+category: how-to
+---
+
+# TiUP 常见运维操作
+
+本文介绍了使用 TiUP 运维 TiDB 集群的常见操作，包括查看集群列表、启动集群、查看集群状态、修改配置参数、关闭集群、销毁集群等。
+
+## 查看集群列表
+
+一个 TiUP 可以管理多个集群，在集群部署完毕之后它会出现在 TiUP 的集群列表里，可以使用 list 命令来查看。
```
Comment (yikeke) [inline how-to/maintain/tiup-operations.md:12]:
```suggestion
TiUP cluster 组件可以用来管理多个 TiDB 集群，在每个 TiDB 集群部署完毕后，该集群会出现在 TiUP 的集群列表里，可以使用 list 命令来查看。
```

File: how-to/maintain/tiup-operations.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+        tidb_servers:
+        - host: 10.0.1.11
+            port: 4000
+            config:
+                log.slow-threshold: 300
+        ```
+
+    参数的格式参考 [TiUP 配置参数模版](https://github.com/pingcap-incubator/tiup-cluster/blob/master/topology.example.yaml)。
+
+    **配置项层次结构使用'.'表示**。
```
Comment (yikeke) [inline how-to/maintain/tiup-operations.md:102]:
```suggestion
    **配置项层次结构使用 `.` 表示**。
```

**Pull request merged**